src/main/resources/application-local.yml

```diff
@@ -4,10 +4,10 @@ feature:
     fetch-sentence-data: true
 
 community-api:
-  base-url: http://localhost:8096
+  base-url: http://localhost:8090
```

Comment:
The 8096 / 8097 values are there to match the docker-compose for running services locally. Do you want these permanently changed ?

Reply:
Ah apologies, had it running against Wiremock. Might create another config for that but will revert this